scout/constants/case_tags.py

```diff
@@ -73,6 +73,10 @@
         "label": "Carrier",
         "description": "Assay performed to identify carrier status found variant present",
     },
+    "incidental": {
+        "label": "Incidental",
```

Comment:
What about IFs? Because the Incidental seems referred to the case if it's written like this

Reply:
After some discussion we keep it - otherwise we'd have to do Diagnostic Finding and Provisional Finding - but move it up one step on the list.